frontend/src/pages/articles/my-articles.tsx

```diff
@@ -1,12 +1,54 @@
+import {
+  useAppDispatch,
+  useAppSelector,
+  useCallback,
+  useEffect,
+} from '~/libs/hooks/hooks.js';
+import { type UserDetailsResponseDto } from '~/packages/users/users.js';
+import { actions as articlesActions } from '~/slices/articles/articles.js';
+
 import { ArticleCard } from './components/components.js';
-import { MOCKED_ARTICLES, MOCKED_USER } from './libs/constants.js';
-
-const MyArticles: React.FC = () => (
-  <>
-    {MOCKED_ARTICLES.map((article) => (
-      <ArticleCard key={article.id} article={article} user={MOCKED_USER} />
-    ))}
-  </>
-);
+import { MOCKED_REACTIONS, MOCKED_TAGS } from './libs/constants.js';
+
+const MyArticles: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { articles, user } = useAppSelector((state) => ({
+    articles: state.articles.articles,
+    user: state.auth.user,
+  }));
+
+  const userDetails = {
+    firstName: user?.firstName,
+    lastName: user?.lastName,
+  } as UserDetailsResponseDto;
```

Comment:
```suggestion
  const userDetails = {
    firstName: user!.firstName,
    lastName: user!.lastName,
  };
```

Reply:
In such case it throws an error: Forbidden non-null assertion.eslint[@typescript-eslint/no-non-null-assertion](https://typescript-eslint.io/rules/no-non-null-assertion)

Reply:
Can I leave this as it is or change the eslint configuration?